Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
```

Comment:
If you're going to add modulo to the valid operations, it should be listed as one of the operations the user can input

Reply:
Everything in run_operation looks good, but it would be good to check the bounds of the inputs to make sure the user inputs valid operations and numbers

Reply:
agreed, it should an option for the user, and making sure the program doesn't allow invalid inputs.